bff-node/src/config/passport.ts

```diff
@@ -0,0 +1,158 @@
+import { Issuer, Strategy as OpenIDStrategy, TokenSet } from 'openid-client';
+import './env';
+import { IDPortenProfile } from '../types/types';
+import { Profile } from '../entities/Profile';
+import { ProfileRepository, SessionRepository } from '..';
+import passport from 'passport';
+import { readCookie } from '../util/sessionUtils';
+import { SessionData } from '../entities/SessionData';
+
+export const initPassport = async () => {
+  if (!SessionRepository) throw new Error('SessionRepository not initialized');
+  if (!ProfileRepository) throw new Error('ProfileRepository not initialized');
+  if (!process.env.ISSUER_URL) {
+    console.error('No issuer url found in environment variables, exiting');
```

Comment:
A suggestion would be to separate the logger into a file `logger.ts` with something like this:

```
export default {
   info: winston.info,
   error: winston.error,
   ...
}
```

This way we could centralize logging and use a tool like `winston` or `pino` in order to get better formatting and to output logs to a bug-tool or format the logs as `json` when running as a container

Can just map it to console.log and console.error for now tho!

Reply:
Enig! :) Men tar det som en separat taks 👍